coalib/settings/SectionFilling.py

```diff
@@ -29,12 +30,20 @@ def fill_settings(sections, acquire_settings, log_printer):
 
     for section_name, section in sections.items():
         bear_dirs = section.bear_dirs()
-        bears = list(section.get('bears', ''))
-        section_local_bears, section_global_bears = collect_bears(
-            bear_dirs,
-            bears,
-            [BEAR_KIND.LOCAL, BEAR_KIND.GLOBAL],
-            log_printer)
+        if hasattr(section, 'aspects') and section.aspects is not None:
```

Comment:
The attrib checking is because of https://github.com/coala/coala/pull/4397#discussion_r128897077. Some of tests is calling `fill_settings` directly without passing `gather_configuration` and that make `section.aspects` undeclared.